mrgf/acapy.py

```diff
@@ -23,7 +23,7 @@ async def connection_to_principal(
         raise TypeError("connection must be connection id or ConnRecord")
 
     conn_record = cast(ConnRecord, conn_record)
-    framework = session.inject(GovernanceFramework, required=False)
+    framework = session.inject(GovernanceFramework)
```

Comment:
We should make this an `inject_or` to keep the same semantics

Reply:
I did not know that `inject_or` was a thing, I'll take a look at that now.

Reply:
Yep, I'm responsible for this particular break lol since I added inject_or to make typing better

Reply:
I've since switched it over.